packages/rest-typings/src/v1/customUserStatus.ts

```diff
@@ -35,14 +36,55 @@ const CustomUserStatusListSchema = {
 
 export const isCustomUserStatusListProps = ajv.compile<CustomUserStatusListProps>(CustomUserStatusListSchema);
 
+type CustomUserStatusCreateProps = { name: string; statusType?: UserStatus };
```

Comment:
The validation expects statuses not 100% equal to this type as you can see here: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/user-status/server/methods/insertOrUpdateUserStatus.ts#L67

Are we good with this change?

Reply:
I've created a new `CustomizableUserStatus` type/enum to fix this